gnucash/import-export/qif-imp/assistant-qif-import.c

```diff
@@ -94,8 +94,10 @@ enum qif_trans_cols
 {
     QIF_TRANS_COL_INDEX = 0,
     QIF_TRANS_COL_DATE,
+    QIF_TRANS_COL_DATE_INT64,
     QIF_TRANS_COL_DESCRIPTION,
     QIF_TRANS_COL_AMOUNT,
+    QIF_TRANS_COL_AMOUNT_DOUBLE,
```

Comment:
Perhaps add a developer comment here to explain why you chose a double instead of a gnc_numeric. I suppose (after some thought) this is because you can store a double directly in a list store where you can't store a gnc_numeric directly.

Reply:
If it's just for sorting numerically it should be OK. For that purpose converting to a common denominator (100 or 1000 seem reasonable) is an alternative, not sure which would be faster nor if it would matter.

Reply:
I will add a comment to fact it is only used for the sorting, I don't think speed is an issue here.